src/main/java/com/nutomic/syncthingandroid/activities/DeviceActivity.java

```diff
@@ -82,11 +90,21 @@
 
     private boolean mDeviceNeedsToUpdate;
 
+    private boolean mIsShowingDiscardDialog;
+    private boolean mIsShowingCompressionDialog;
+    private boolean mIsShowingDeleteDialog;
+
+    private Dialog mDeleteDialog;
+    private Dialog mDiscardDialog;
+    private Dialog mCompressionDialog;
+
+
+
```

Comment:
You have a lot of empty lines in some places. Can you get rid of those so you only have one empty line at a time? Means that more code fits on the screen ;)